ctapipe/reco/ImPACT.py

```diff
@@ -18,13 +17,29 @@
 from ctapipe.io.containers import (ReconstructedShowerContainer,
                                    ReconstructedEnergyContainer)
 from ctapipe.reco.reco_algorithms import Reconstructor
-from ctapipe.reco.shower_max import ShowerMaxEstimator
 from ctapipe.utils import TableInterpolator
-from ctapipe import instrument
+from ctapipe.instrument import get_atmosphere_profile_functions
+from scipy.optimize import minimize, least_squares
+from scipy.stats import norm
+
+__all__ = ['ImPACTReconstructor', 'energy_prior', 'xmax_prior']
+
+
+def energy_prior(energy, index=-1):
+
+    return -2 * np.log(np.power(energy, index))
+
+
+def xmax_prior(energy, xmax, width=30):
+
+    x_max_exp = 300 + 93 * np.log10(energy)
```

Comment:
In this case I think we can just use the ShowerMaxEstimator. I forgot this exists, will change it.